components/Molecule/main-banner.html

```diff
@@ -6,7 +6,6 @@
 
```

Comment:
I think this is at version 1.3 now!

Reply:
ok. does every change needs 0.1 increament? should I commit now as 1.4?